lib/ex_unit/test/ex_unit_test.exs

```diff
@@ -265,6 +265,25 @@ defmodule ExUnitTest do
     assert output =~ "2 tests, 0 failures, 2 skipped"
   end
 
+  test "includes test tagged with skip" do
```

Comment:
Instead of an integration test, can you please add a test to lib/ex_unit/test/filters_test.exs?